front/components/assistant/AssistantList.tsx

```diff
@@ -0,0 +1,65 @@
+import { AssistantPreview } from "@dust-tt/sparkle";
+import type {
+  ConversationType,
+  LightAgentConfigurationType,
+} from "@dust-tt/types";
+import { useRouter } from "next/router";
+import React from "react";
+
+interface AssistantListProps {
+  agents: LightAgentConfigurationType[];
+  handleAssistantClick: (
+    agent: LightAgentConfigurationType,
+    conversation?: ConversationType
+  ) => void;
+}
+
+const AssistantList: React.FC<AssistantListProps> = ({
```

Comment:
nit: We mostly don't use this React.FC thing and instead tend to use function. But I guess it doesn't matter much I'm not too much against it.